greenplumpython/func.py

```diff
@@ -61,14 +60,15 @@ def bind(
     ):
         # noqa D400
         """:meta private:"""
-        return FunctionExpr(
+        f = FunctionExpr(
             self._func,
             self._args,
             group_by=group_by,
             dataframe=dataframe,
-            db=db if db is not None else self._db,
             distinct=self._distinct,
         )
+        f._db = db if db is not None else self._db
```

Comment:
assert self._db is not None here?

Reply:
`assert db is not None or self._db is not None`?

Reply:
yes.

Reply:
Done